<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1035</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="addSubjectPushButton">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="subjectLineEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="subjectListWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="tableView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="cornerButtonEnabled">
         <bool>false</bool>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QWidget" name="widget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QPushButton" name="addStudentPushButton">
             <property name="text">
              <string>Add student</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="addTaskPushButton">
             <property name="text">
              <string>Add Task</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="addCalcColumnPushButton">
             <property name="text">
              <string>Add Column</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="addCalcRowPushButton">
             <property name="text">
              <string>Add Row</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="addCalcLayerPushButton">
             <property name="text">
              <string>Add Layer</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="AddStudentWidget" name="addStudentWidget" native="true"/>
          </item>
          <item>
           <widget class="AddTaskWidget" name="addTaskWidget" native="true"/>
          </item>
          <item>
           <widget class="SwapStudentWidget" name="swapStudentWidget" native="true"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="GeneratorWizzard" name="addGeneratorWidget" native="true"/>
          </item>
          <item>
           <widget class="GeneratorEditor" name="generatorEditorWidget" native="true"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="layerListWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1035</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AddStudentWidget</class>
   <extends>QWidget</extends>
   <header>addstudentwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AddTaskWidget</class>
   <extends>QWidget</extends>
   <header>addtaskwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GeneratorWizzard</class>
   <extends>QWidget</extends>
   <header>generatorwizzard.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GeneratorEditor</class>
   <extends>QWidget</extends>
   <header>generatoreditor.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SwapStudentWidget</class>
   <extends>QWidget</extends>
   <header>swapstudentwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
